functionaltest/playwright/components/locators/homelocators.ui.ts

```diff
@@ -0,0 +1,14 @@
+import { Locator, Page } from "@playwright/test"
+
+export class HomeLocators {
+readonly sportsbookbtn : Locator;
+readonly logo: Locator;
+readonly aboutus: Locator;
+readonly whychamsports: Locator;
```

Comment:
We need space hear

Reply:
Based on your suggentions comments updated